### DISCUSSION TITLE
Uploading a background image doesn't work anymore

### DISCUSSION THREAD

**Discussion by abelfubu** [bluecaret/carettab#342] [Help and Support] [answered]
- After the update my background image disappeared and when trying to upload a custom image again the image is never being displayed.

**Comment by bluecaret** [comment]
Thank you for reporting this. I've done testing with several images during development and haven't seen this issue so I'm wondering if it is a problem with the specific image you are using? Have you tried others? If you don't mind, could you send the image to me at dev@bluecaret.com and I can give it a shot and see why that image isn't working. The update should have kept it as well.

**Comment by bluecaret** [comment, marked as answer]
Hello, just letting you know I figured out what the issue is. Can't say I understand why it was an issue, but regardless it works and I've tested your image and it loads just fine now :) I'll be pushing an update soon with a few fixes in it. Keep a look out for the next update in a few days. Thank you!

**Comment by abelfubu** [comment]
Hi!!

Thank you so much!!
Thanks for the hard work and the great extension!

Cheers!

On Sat, 29 Jul 2023 at 05:16, John Hancock ***@***.***> wrote:

> Hello, just letting you know I figured out what the issue is. Can't say I
> understand why it was an issue, but regardless it works and I've tested
> your image and it loads just fine now :) I'll be pushing an update soon
> with a few fixes in it. Keep a look out for the next update in a few days.
> Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bluecaret/carettab/discussions/342#discussioncomment-6580831>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APR4F3CSN77DACQZIFII3VLXSR6AHANCNFSM6AAAAAA2Z2H7MI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
[image: abeldelafuente] ***@***.***> *Abel de la
Fuente*

Fullstack Web developer | Typescript | Angular
<https://wa.me/617366135> <https://linkedin.com/in/in/abelfubu>
<https://github.com/abelfubu>
+34 617 36 61 35Barcelona, Spain
